### DISCUSSION TITLE
Permit to enable JIT AND get all CSS from Tailwindcss classes in developement mode

### DISCUSSION THREAD

**Discussion by 8Thomas8** [tailwindlabs/tailwindcss#4025] [Ideas]
Hello.

It will be usefull to permit to get the +  of JIT and to get all the pregenerated css from Tailwindcss, in dev mode, as an option. So we can continue to test classes in browsers devtools.

**Comment by foxtrotcz** [comment]
Maybe would be better to do it like WindiCSS and use MutationObserver so classes are generated even if you add them in devtools.
https://windicss.org/integrations/vite.html#design-in-devtools
